app/views/claims/bank_details.html.erb

```diff
@@ -36,6 +44,16 @@
               autocomplete: "off",
               "aria-describedby" => "account-number-hint" %>
```

Comment:
Interestingly the draft pattern for bank account number says the number can be between 6 to 8 digits, do we need to update our validation rules?